fe/fe-core/src/main/java/org/apache/doris/planner/TableFunctionNode.java

```diff
@@ -118,7 +118,7 @@ public void projectSlots(Analyzer analyzer, SelectStmt selectStmt) throws Analys
         for (TupleId tupleId : inputTupleIds) {
             TupleDescriptor td = analyzer.getTupleDesc(tupleId);
             for (SlotDescriptor sd : td.getSlots()) {
-                if (!outputSlotIds.contains(sd.getId())) {
+                if (!outputSlotIds.contains(sd.getId()) && !sd.getColumn().isDeleteSignColumn()) {
```

Comment:
Need to make sure that this change will not go wrong in the case of line storage.
Whether be has set null for delete column in row engine.